### ISSUE TITLE
Multiple link download ask one by one the download location

### ISSUE THREAD

**Issue by technofab** [downthemall/downthemall#168]
**Desktop (please complete the following information):**
 - OS: [e.g. Windows 10, macOS, Linux (distribution, desktop environment)]

MACOS MOJAVE

 - Browser and version: [e.g. Firefox 42, Chrome 70, Opera 15, Seamonkey 2.16]

Latest Chrome

 - DownThemAll! version: [e.g. 4.2, 3.0, latest]

Latest

**Describe the bug**
Multiple link download ask one by one the download location

**To Reproduce**
I select some link there point to images
Click on save selection or save link with downthemall
Select where to download and start the mass download.
For every image ask where to download

**Expected behavior**
It must not ask every download where download.

**Comment by nmaier** [comment]
This is a quirk in Chrome itself. As long as you have the "Ask for every download" checkbox enabled in the Chrome preferences, it will do just that even for extensions-initiated downloads. I have found no way around it.

The only way to make chrome less annoying with file prompts is to disable said preference.

**Comment by technofab** [comment]
You are right. I set Chrome you wrote and works. Until you find a workaround (if possible) in the options can you write this? I hope helps someone else. Thank you for your help.
